recipes/libflint/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+source activate "${CONDA_DEFAULT_ENV}"
+
+chmod +x configure
+
+echo "int flint_test_multiplier(){return 1;}" > test_helpers.c
```

Comment:
I've explained this in a comment. It reduces the number of random tests in flint.